text/0072-online-bulk-load-for-rawkv.md

```diff
@@ -0,0 +1,103 @@
+# RFC: Online Bulk Load for RawKV
+
+- RFC PR: https://github.com/tikv/rfcs/pull/72
+- Tracking Issue: https://github.com/tikv/tikv/issues/10563
+- [Proposal: TiKV Online Bulk Load (Chinese)](https://docs.google.com/document/d/1JggGwmOIMqinz_Q7QkIDHzwS--K_BgJopaxrB8Pzpaw/edit#)
+
+## Summary
+
+Let RawKV support Online Bulk Load.
```

Comment:
1. this RFC only described the solution on the tikv side. Could you also supplement the whole solution based on the bulk load APIs? for example, how to provide a pluggable bulk load spark job that enables users to add custom transmission between source data to kv pairs?

2. could you describe how to test the ingesting speed and the performance impact on the online services? and how to verify that the ingested data is correct compared with the source data stored in files?

Reply:
sure

Reply:
https://github.com/marsishandsome/rfcs/blob/feature/bulk-load/text/0072-online-bulk-load-for-rawkv.md#how-to-measure-the-performance-impact-to-the-online-services

https://github.com/marsishandsome/rfcs/blob/feature/bulk-load/text/0072-online-bulk-load-for-rawkv.md#checksum

https://github.com/marsishandsome/rfcs/blob/feature/bulk-load/text/0072-online-bulk-load-for-rawkv.md#checksum